.github/workflows/release.yml

```
@@ -0,0 +1,192 @@
+        run: |
+          mkdir -p release
+          cp proxy/build/libs/proxy-*.jar release/ 2>/dev/null || true
+          cp api/build/libs/api-*.jar release/ 2>/dev/null || true
+          cp bridge/build/libs/bridge-*.jar release/ 2>/dev/null || true
+          # Remove javadoc and sources jars from release
+          rm -f release/*-javadoc.jar release/*-sources.jar 2>/dev/null || true
+          echo "Release artifacts:"
+          ls -la release/
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)